web_tree_background_color/static/src/js/list_view.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Leonardo Donelli (donelli@webmonks.it)
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_tree_background_color.stuff', function (require) {
+    "use strict";
+
+    var ListView = require('web.ListView');
+    var session = require('web.session');
+
+    ListView.include({
+
+        init: function() {
+            var res = this._super.apply(this, arguments);
+            this.bg_decoration = null;
```

Comment:
mention what this var is going to contain (eg: list of something, mapping expr: rule, or similar) in a comment above it